source/core.py

```diff
@@ -611,53 +675,10 @@ def main():
 		log.debugWarning("Slow starting core (%.2f sec)" % timeSinceStart)
 		# Translators: This is spoken when NVDA is starting.
 		speech.speakMessage(_("Loading NVDA. Please wait..."))
-	import wx
-	import six
-
-	# Disables wx logging in secure mode due to a security issue: GHSA-h7pp-6jqw-g3pj
-	# This is due to the wx.LogSysError dialog allowing a file explorer dialog to be opened.
-	wx.Log.EnableLogging(not globalVars.appArgs.secure)
-
-	log.info("Using wx version %s with six version %s"%(wx.version(), six.__version__))
-	class App(wx.App):
-		def OnAssert(self,file,line,cond,msg):
-			message="{file}, line {line}:\nassert {cond}: {msg}".format(file=file,line=line,cond=cond,msg=msg)
-			log.debugWarning(message,codepath="WX Widgets",stack_info=True)
-
-		def InitLocale(self):
-			"""Custom implementation of `InitLocale` which ensures that wxPython does not change the locale.
-			The current wx implementation (as of wxPython 4.1.1) sets Python locale to an invalid one
-			which triggers Python issue 36792 (#12160).
-			The new implementation (wxPython 4.1.2) sets locale to "C" (basic Unicode locale).
-			While this is not wrong as such NVDA manages locale themselves using `languageHandler`
-			and it is better to remove wx from the equation so this method is a No-op.
-			This code may need to be revisited when we update Python / wxPython.
-			"""
-			pass
 
+	import wx

```

Comment:
Given that you're importing wx in 3 different places, including in the main execution flow, can we just put the single import up at the top of the core.py file? If there's no way to avoid loading it (which it looks like from def main()), we can simplify the imports and remove the TYPE_CHECKING logic.

Reply:
I believe we are intentionally avoiding the wx import until further down. Dependency importing is incredibly order specific in nvda.pyw and core.py